rebar.config.erl

```diff
@@ -268,3 +268,27 @@ str(B) when is_binary(B) -> unicode:characters_to_list(B, utf8).
 
 erl_opts_i() ->
     [{i, "apps"}] ++ [{i, Dir}  || Dir <- filelib:wildcard(filename:join(["apps", "**", "include"]))].
+
+dialyzer(Config) ->
+    {dialyzer, OldDialyzerConfig} = lists:keyfind(dialyzer, 1, Config),
+
+    AppsToExclude = case os:getenv("DIALYZER_ANALYSE_APP") of
+        false ->
+            [];
+        Value ->
```

Comment:
we should support comma separated values.
e.g. `DIALZYER_ANALYSE_APP="emqx_auth_mnesia,emqx_auth_jwt"`